### PR TITLE
merge fuse image to mount & update 1.2 Dockerfile to go1.20

### PR DESCRIPTION
**Pull request by zwwhdls**
1. merge fuse image to mount 
2. rename ce.juicefs-1.2.Dockerfile to ce.juicefs.Dockerfile 
3. rename ce.juicefs.Dockerfile to ce.juicefs-1.1.Dockerfile
4. update 1.2 Dockerfile to go1.20

### CODE REVIEW

File: docker/ee.juicefs.Dockerfile [hunk 1]
```diff
@@ -42,3 +42,6 @@ RUN jfs_mount_path=${JFS_MOUNT_PATH} && \
     chmod +x ${jfs_mount_path} && cp juicefs.py ${JUICEFS_CLI} && chmod +x ${JUICEFS_CLI}
 
 RUN /usr/bin/juicefs version
+
+ENV K8S_VERSION v1.14.8
+RUN curl -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/${TARGETARCH}/kubectl && chmod +x /usr/local/bin/kubectl
```
Comment (zxh326) [inline docker/ee.juicefs.Dockerfile:47]: why we need `kubectl` in mount image

Comment (zwwhdls) [reply docker/ee.juicefs.Dockerfile:47]: It's need in fuse image which is for Fluid. This PR merged the two in order to better maintain the image in the future.

**Review by zxh326 — APPROVED** [review]
I’m concerned that this might cause the mount image size to keep growing

**Comment by zwwhdls** [comment]
There will be only a dozen M of growth, and no other unnecessary packages will be added.

**Pull request merged**